### PR TITLE
flink_table: dont use cpu_in_mb

### PR DESCRIPTION
**Pull request by mhoffm-aiven**
# What changed, and why it matters

This PR gets rid of the `cpu_in_mb` column in the flink table example.

### CODE REVIEW

File: docs/tools/cli/service/flink.rst [hunk 1]
```diff
@@ -255,4 +255,4 @@ An example of ``avn service flink job list`` output:
 
   ID                                STATUS
   ================================  =======
-  b63c78c70033e00afa84de9029257e31  RUNNING
```
Comment (laysauchoa) [inline docs/tools/cli/service/flink.rst:258]: hey, could you please explain why was this changed?

**Comment by laysauchoa** [comment]
oki, `main` is fixed so it should work if you rebase.

File: docs/tools/cli/service/flink.rst (hunk 1 above)
Comment (mhoffm-aiven) [reply docs/tools/cli/service/flink.rst:258]: i have no idea why it shows up, both lines look identical to me, let me compare whitespaces really quick

Comment (mhoffm-aiven) [reply docs/tools/cli/service/flink.rst:258]: whitespaces look identical too in both branches

**Comment by laysauchoa** [comment]
For now, we will wait to maybe a future fix related to this: https://github.com/aiven/aiven-core/pull/18940 
The percentage shall be reviewed first in a different PR by @mhoffm-aiven

**Review by laysauchoa — APPROVED** [review]
lgtm, thanks @mhoffm-aiven
